internal/systemd/systemd_test.go

```diff
@@ -0,0 +1,84 @@
+package systemd
```

Comment:
I'd like to suggest adding some failure state tests if not to difficult.

Reply:
Just FYI: These test did pass when I ran them.

Reply:
I think, to properly test for errors, we'll need to refactor the error structure
in the package somewhat. It's a good thing to do, but I'm not sure it's worth
doing now. Might be better to do as a smaller refactor afterwards.

Reply:
Sure, I'm OK with addressing this as a follow-up PR